index.bs

```diff
@@ -263,6 +263,35 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">New features should not break existing content</h3>
+
+When making changes to web APIs, consider the impact on existing web content.
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing user agents.
```

Comment:
```suggestion
It should be possible to process existing web content in current user agents and
get results that are compatible with preexisting expectations of users and authors.
This will generally match the behavior of user agents prior to the introduction of the new feature.
```